src/github/client/github-client.ts

```diff
@@ -26,5 +29,22 @@ export class GitHubClient {
 			this.restApiUrl = `${baseUrl}/api/v3`;
 			this.graphqlUrl = `${baseUrl}/api/graphql`;
 		}
+
+		const proxyConfig = this.getProxyConfig(this.restApiUrl);
+
+		this.axios = axios.create({
+			baseURL: this.restApiUrl,
+			transitional: {
+				clarifyTimeoutError: true
+			},
+			... proxyConfig
+		});
 	}
+
+	private getProxyConfig = (baseUrl: string): Partial<AxiosRequestConfig> => {
+		if (baseUrl.includes("github.internal.atlassian.com")) {
```

Comment:
I feel a great disturbance in the force...

should we be ignoring the proxy altogether for all `atlassian.com` or even `atlassian.net` urls so that it gets routed internally instead of going to the internet first?

Reply:
made it so that it matches everything that ends with atlassian.com, now